ui/src/App.css

```diff
@@ -1,3 +1,7 @@
 .app {
   margin: 20px;
 }
+
+.legal-disclaimer {
```

Comment:
opt BTW, Brett said they were doing inline CSS for Saturn UI. I'm starting to do that, since it's annoying to switch back and forth between .js and .css. (And it's harder to delete obsolete styles in .css.) For this and future PRs, feel free to use [inline CSS](https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822).

Reply:
Done.

Reply:
Revert this